src/frontend/main/Overview/index.tsx

```diff
@@ -34,226 +32,163 @@ const Table = styled.table`
 
   th,
   td {
-    padding: 10px 4px;
+    padding: 6px 4px;
+    line-height: 1.1;
+    vertical-align: top;
+  }
 
-    @media (min-width: 450px) {
-      width: 140px;
-    }
+  th:nth-child(1) {
+    width: calc(2ch + 16px);
+  }
+
+  th:nth-child(3) {
+    /* ch = relative width of a character "0", seems to
+     * fit better for adjusting number column width
+     * */
+    width: calc(17ch + 20px);
+    padding-right: 16px;
+    padding-left: 4px;
+    text-align: right;
   }
 
-  th:nth-child(1),
   td:nth-child(1) {
-    padding-left: 16px;
-    width: 7%;
+    padding-left: 0;
+    text-align: right;
   }
 
-  th:nth-child(3),
   td:nth-child(3) {
-    width: 25%;
     padding-right: 16px;
-    text-align: right;
+    padding-left: 4px;
+    text-align: left;
   }
 
-  tr:nth-child(2n) {
-    background: #f1f1f1;
+  tr:nth-child(2n + 1) {
+    background: ${({ theme }) => theme.lightGrey};
   }
 `;
 
 const Container = styled.div`
-  max-width: 648px;
+  max-width: ${({ theme }) => theme.mobileWidth}px;
   margin: 0 auto;
-  padding: 24px;
-`;
+  padding: 0 10px;
 
-const Heading = styled.div`
-  font-size: 22px;
-  font-weight: bold;
-  margin: 30px 0 5px 0;
-`;
+  @media (max-width: ${({ theme }) => theme.mobileWidth}px) {
+    padding: 0;
+  }
 
-const H1 = styled.h1`
-  font-size: 32px;
-  font-weight: bold;
-  margin-bottom: 5px;
-`;
+  h1 {
+    font-size: 32px;
+    margin-bottom: 5px;
+    margin-top: 24px;
+    line-height: 1.1;
+  }
 
-const P = styled.p`
-  margin-top: 0;
+  h2 {
+    font-size: 22px;
+    margin: 45px 0 5px 0;
+    line-height: 1.1;
+  }
+  p {
+    margin-top: 0;
+  }
 `;
 
 const NumberText = styled.span`
   font-size: 28px;
   font-weight: bold;
 `;
 
-const Div = styled.div`
+const SubHeadline = styled.div`
   font-size: 18px;
   font-style: italic;
-  margin: 10px 0;
-`;
-
-const SpanBold = styled.span`
-  font-weight: bold;
+  margin-bottom: 10px;
 `;
 
 const TableHead = styled.thead`
-  background: ${props => props.theme.lightGrey};
+  tr:first-child {
+    background: ${props => props.theme.white};
+  }
 
   th {
     padding: 10px 4px;
-    font-weight: bold;
     font-size: 14px;
   }
 `;
 
-const SelectionContainer = styled.div`
-  display: flex;
-  align-items: center;
-  p {
-    margin: 0 10px;
+const MobilePadding = styled.div`
+  width: 100%;
+  @media (max-width: ${({ theme }) => theme.mobileWidth}px) {
+    padding: 0 16px;
```

Comment:
We're following mobile first approach, so the default value should be for mobile, and the overridden for bigger breakpoints when needed, with min-width breakpoint.